todo_app/flask_config.py

```diff
@@ -1,6 +1,50 @@
-import os
+import os, requests
 
 
+def get_access_token(CLIENT_ID: str, CLIENT_SECRET: str, request_token: str) -> str:
```

Comment:
I'm not really sure why these are in a flask_config file, would this make more sense in their own `github_oauth_flow_helpers.py` or something? That's a bit of a mouthful so better names definitely encouraged!